install/installer/pkg/components/server/configmap.go

```diff
@@ -237,6 +237,10 @@ func configmap(ctx *common.RenderContext) ([]runtime.Object, error) {
 			// default limit for all cloneURLs
 			"*": 50,
 		},
+		WorkspaceClasses: WorkspaceClasses{
+			Default:              "g1-standard",
```

Comment:
Ok, now I'm absolutely confused :confused:
 - I know `gitpodio-internal-xl` is a thing: is those one of the old values?
 - I recall discussions where we were talking about "Standard" and "XL" (I bet those are the descriptions, right? But for which classes??)
 - I see the following values thrown around in code:
   - ""
   - "default"
   - "standard"
   - "g1-large"
   - "g1-standard"

:exploding_head: 

@Furisto Could you add an explanatory table to the top of the PR, please?

And for the code in this PR: We shoudl remove _all_ usages of concrete string values from `server` code, and push that variance into the `workspaceClasses` config of this shape (or similar):

```
// Types
type WorkspaceClassesConfig = [WorkspaceClassConfig]
interface WorkspaceClassConfig {
  // The technical string we use to identify the class with internally
  workspaceClass: string;
  
  // Is the "default" class. The config is validated to only every have exactly _one_ default class.
  isDefault: boolean;
 
  // The string we display to users in the UI
  description: string;

  // Whether or not to:
  //  - offer users this Workspace class for selection
  //  - use this class to start workspaces with. If a user has a class marked like this configured and starts
  deprecated: boolean;

  // The price for this workspace class in "credits per minute"
  creditsPerMinute: number;
}
```
This should be sufficient to push all the string values into that. :lotus_position:

Reply:
Has been resolve :rocket: